src/components/Breadcrumb/Breadcrumb.less

```diff
@@ -0,0 +1,43 @@
+.lucid-Breadcrumb {
```

Comment:
i always thought this was better, too, but i thought we were trying to avoid deep nesting